EnglishWebRoot/school/pubbly_engine/latest/js/app/_AppModifications.js

```diff
@@ -1,19 +1,55 @@
 // Main class for all android/ios app modifications (assuming cordova)
+var endTime, startTime;
+function Calculate_time() {
+
+    var url = window.location.pathname.split("/");
+    url.pop(); // index.html
+    var book = url.pop();
+    var subject = url.pop();
+    var Module_name = url.pop();

```

Comment:
Use proper naming conventions for the variable name

Reply:
Changed variable names according to naming conventions.